packages/kbn-esql-validation-autocomplete/src/validation/validation.test.ts

```diff
@@ -9309,9 +9307,7 @@ describe('validation logic', () => {
           'EVAL does not support function min',
         ]);
 
-        testErrorsAndWarnings('from a_index | stats min(booleanField)', [
-          'Argument of [min] must be [number], found value [booleanField] type [boolean]',
-        ]);
+        testErrorsAndWarnings('from a_index | stats min(booleanField)', []);
```

Comment:
Let's run `yarn make:tests` to create a new error case now that boolean is acceptable.

Reply:
I didnt know about that